docs/metrics.md

```diff
@@ -0,0 +1,40 @@
+# EDC extensions for instrumentation with Micrometer
+
+EDC provides extensions for instrumentation with the [Micrometer](https://micrometer.io/) metrics library to automatically collect metrics from the host system, JVM, and frameworks and libraries used in EDC (including OkHttp, Jetty, Jersey and ExecutorService).
+
+## Micrometer Extension
+
+This extension provides support for instrumentation for some core EDC components:
+- JVM metrics
+- [OkHttp](https://square.github.io/okhttp/) client metrics
+- [ExecutorService](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ExecutorService.html) metrics
+
+### Instrumenting ExecutorServices
```

Comment:
make clearer that this is independent of micrometer, and should be used in all cases.

and that a no op instrumentation is used in micrometer is not used.